### ISSUE TITLE
[Adaptive cards]: Screen reader announces extra information as 'App bar' with all the header controls.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7826]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Browser: Edge Dev (106.0.1356.0)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2206.2001
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ windows+ enter key.
3. Navigate to the header control through tab key.
4. Observe the issue.

**Actual Result:**
On all the header controls such as on 'New card', 'Save file', 'Open file' and on 'Host config', screen reader announces extra information as app bar.

**Expected Result:**
Screen reader should not announce extra information as 'App bar' with all the header controls.

**Observation:**
Same issue repro with NVDA and JAWS.

**User Impact:**
Person with cognitive disability will get impacted if extra information is announced by the screen reader and it appears to be too verbose to the user which might confused the user or user will not be able to interact with the content properly.

**MAS Reference link:**[
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

**“Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/104483005/189741675-63ec4f34-d4f2-4ac8-a040-0543d7435c28.mp4

### Card JSON

```JSON
NA
```

**Comment by Rishikesh6521** [comment]
Closing this bug as this bug has been active for more than 2 years logging a same new bug and tracking the bug through #9236
